docs/topics/resources/spring-boot/reactive-examples/example-http-service-authenticated-webflux-pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>dev.snowdrop</groupId>
+    <artifactId>vertx-spring-boot-samples</artifactId>
+    <version>0.0.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>vertx-spring-boot-sample-http-security</artifactId>
+
+  <dependencies>
+<!-- tag::http-service-authenticated-pom-dependencies[] -->
+    <dependency>
+      <groupId>dev.snowdrop</groupId>
+      <artifactId>vertx-spring-boot-starter-http</artifactId>
+      <version>${project.version}</version>
```

Comment:
Same as in the simple HTTP sample. No need for the version because `spring-boot-bom` is expected by the documentation.